lib/tasks/fix_coord_mismatch.rake

```diff
@@ -6,15 +6,14 @@ task fix_coord_mismatch: :environment do
   results = Institution.approved_institutions(version)
```

Comment:
This rake file is getting a bit large. It would make sense to extract the logic into a class so that it could be tested easier.